packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3303,7 +3303,7 @@ export class ContainerRuntime
 				"Pending ops during summarization",
 				"submitSummary",
 				undefined,
-				{ pendingMessages: this.pendingMessagesCount },
+				{ count: this.pendingMessagesCount },
```

Comment:
I had kept `pendingMessages` because other telemetry in container runtime use that name for logging `this.pendingMessagesCount`

Reply:
Then maybe we should update `new RetriableSummaryError` below - I was trying to match that